expression/builtin_op.go

```diff
@@ -403,6 +407,8 @@ func (c *isTrueOrFalseFunctionClass) getFunction(ctx sessionctx.Context, args []
 		case types.ETInt:
 			sig = &builtinIntIsTrueSig{bf}
 			sig.setPbCode(tipb.ScalarFuncSig_IntIsTrue)
+		default:
+			panic("unexpected types.EvalType")
```

Comment:
it's better to return an error, rather than panic.

Reply:
This switch has covered all cases, I think panic is ok at the unreachable path.

Reply:
It's only covered all the cases for now. If we support more evaluate types in the future, it may lead to a panic and cause the server to out of service.